src/index.html

```diff
@@ -4,9 +4,342 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+    <main class = "main">
+      <div class = "card" data-qa="card">
+        <!-- #region card -->
+        <img
+        src="./images/imac.jpeg"
+        alt="A1419-iMac_image"
+        class="card__image"
+      >
+        <div class = "card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class = "card__code">Product code: 195434</div>
+        <!-- #endregion -->
+        <!-- #region stars -->
+        <div class = "card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class = "card__stars-text">
+            <span>Reviews: 5</span>
+          </span>
+        </div>
+        <!-- #endregion stars -->
+
+        <div class = "card__price">
+          <span class="card__price-title">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <div class="card__button" data-qa="card-hover">
+          <span class="card__button-text">Buy</span>
+        </div>
```

Comment:
```suggestion
        <a class="card__button-text" data-qa="card-hover">Buy</a>
```
Combine this two classes in one